TESTING.md

```diff
@@ -1,47 +1,83 @@
-This cookbook uses a variety of testing components:
-
-- Unit tests: [ChefSpec](https://github.com/acrmp/chefspec)
-- Integration tests: [Test Kitchen](https://github.com/opscode/test-kitchen)
-- Chef Style lints: [Foodcritic](https://github.com/acrmp/foodcritic)
-- Ruby Style lints: [Rubocop](https://github.com/bbatsov/rubocop)
-
-Prerequisites
+Cookbook TESTING doc
+====================
+Testing Prerequisites
+---------------------
+Chef cookbooks require either a working ChefDK installation set as your system's default ruby or Ruby 2.0+ with bundler installed. Using ChefDK provides a consistent Ruby install, and is the suggested method. ChefDK can be downloaded at https://downloads.chef.io/chef-dk/
+
+To ensure all gems are updated to their latest releases run `bundle install; bundle update` before running any Rake testing tasks.
+
+Integration testing relies on Hashicorp's [Vagrant](https://www.vagrantup.com/downloads.html) and
+local virtualization system Oracle's [Virtualbox](https://www.virtualbox.org/wiki/Downloads), which must be installed first.
+
+You can also use Vagrant with [Parallels Desktop for Mac](http://www.parallels.com/products/desktop/)
+(with ["vagarnt-parallels"](https://github.com/Parallels/vagrant-parallels) plugin)
+
+Rakefile
+--------
+The Rakefile ships with a number of tasks, each of which can be ran
+individually, or in groups. Typing "rake" by itself will perform style
+checks with Rubocop and Foodcritic, ChefSpec with rspec, and
+integration with Test Kitchen using the Vagrant driver by
+default. Alternatively, integration tests can be ran with Test Kitchen
+cloud drivers.
+
+```
+$ rake -T
+rake integration:vagrant      # Run Test Kitchen with Vagrant
+rake spec                     # Run ChefSpec examples
+rake style                    # Run all style checks
+rake style:chef               # Run Chef style checks
+rake style:ruby               # Run Ruby style checks
+rake style:ruby:auto_correct  # Auto-correct RuboCop offenses
```

Comment:
Yes. It is defined in `rubocop/rake_task`
https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/rake_task.rb